workitem/event/event_repository_blackbox_test.go

```diff
@@ -379,6 +379,14 @@ func (s *eventRepoBlackBoxTest) TestList() {
 		assert.Equal(t, 2, c)
 	})
 
+	s.T().Run("reorder event shouldn't be logged", func(t *testing.T) {
+		fxt := tf.NewTestFixture(t, s.DB, tf.WorkItems(2))
+		s.wiRepo.Reorder(s.Ctx, fxt.Spaces[0].ID, workitem.DirectionBelow, &fxt.WorkItems[0].ID, *fxt.WorkItems[1], fxt.Identities[0].ID)
```

Comment:
Can you please create 3 events and then move item with ID 1 to down or up? I don't know straight away if item 0 is at the top or the bottom of a list. And I bet there will not be an event if the order has no effect, say because the item is already at the very end of the list.

Reply:
I hope this makes it more clear https://github.com/fabric8-services/fabric8-wit/pull/2196/commits/74feeca998526be7ec74166dbc5af4de120cd12b